src/libs/elektra/plugin.c

```diff
@@ -413,7 +413,8 @@ static int elektraVersionGet (Plugin * handle ELEKTRA_UNUSED, KeySet * returned,
 	KeySet * info = elektraVersionKeySet ();
 	keySetMeta(info->array[0], "restrict/write", "1");
 	keySetMeta(info->array[0], "restrict/remove", "1");
-	for (size_t i = 1; i < info->size; i++) {
+	for (size_t i = 1; i < info->size; i++)
```

Comment:
👋 Hi Dominic, I would really recommend you install [`clang-format`](https://clang.llvm.org/docs/ClangFormat.html#clangformat) and format the file with something like

```sh
clang-format -i src/libs/elektra/plugin.c
```

, the script [`reformat-source`](https://master.libelektra.org/scripts/reformat-source), or something similar. Formatting the file by hand is really error prone, and will most probably not be a fun experience.

Reply:
Thank you for helping! We should have similar text as output in the build server, see  #2336

Reply:
Thank you for the hint! I did not know about this script.